ansible-st2/roles/arteria_node/tasks/core_tasks.yml

```diff
@@ -89,3 +88,16 @@
     name: supervisord
     state: started
 
+- name: ensure /var/log/arteria dir exists
+  file:
+    state: directory
+    path: /var/log/arteria
+    owner: "{{ arteria_user }}"
+
+- name: ensure /etc/arteria dir exists
+  file:
+    state: directory
+    path: /etc/arteria
+
+- name: disable iptables
```

Comment:
Maybe we want to make this a variable for the playbook? Mostly to make sure that it's explicit that this is something that will happen?